docs/index.rst

```diff
@@ -3,6 +3,18 @@
 Installing using Docker
 =======================
 
+.. toctree::
+    :maxdepth: 2
+    :hidden:
+
+    quickstart
+    security/index
+    configuration
+    operations/index
+    tutorials/index
+    development
+    contributing
```

Comment:
was the move of the `contributing.rst` file intentional? where it is, i don't think this reference will work?

Reply:
yeah, intentional. contributing is usually a GH-specific piece of content. and i think it does work, see [this link](https://github.com/confluentinc/cp-docker-images/blob/joel-hamill/fix-nav/contributing.rst). did you mean it wouldn't work because it's `.rst`?

Reply:
i meant that i think the paths in the TOC are relative to the current directory, but since contributing isn't in the `docs/` directory anymore I didn't think this would actually get linked properly.

in any case, i can wait until this is no longer WIP to review :)